bin/serial-number

```diff
@@ -7,11 +7,17 @@ var fs = require('fs');
 
 var serialNumber = require('../');
 
+var cmdprefix = '/usr/sbin/';
 if (process.argv[process.argv.length - 1] === '--uuid') {
 	serialNumber.preferUUID = true;
+    if (process.argv[process.argv.length - 2].indexOf('--cmdprefix') > -1) {
```

Comment:
With my suggestion the CLI syntax would become `--cmdPrefix /usr/sbin/` instead of `--cmdPrefix=/usr/sbin/`